docs/devguide/contributing.asciidoc

```diff
@@ -51,6 +51,8 @@ used for development is Go {go-version}.
 The location where you clone is important. Please clone under the source
 directory of your `GOPATH`. If you don't have `GOPATH` already set, you can
 simply set it to the `go` directory in your home (`export GOPATH=$HOME/go`).
+Don't forget setting up new PATH variable for correct mage execution:
```

Comment:
This not a requirement levied by `mage`. It’s a pretty standard requirement for all Go development. 
https://golang.org/doc/code.html#GOPATH

Can you reword the statement.

Reply:
No problem. I just see to manual a do step  by step my new module. This is once problem, that i meet on my way.